### ISSUE TITLE
How long does it to train？

### ISSUE THREAD

**Issue by kuiran** [chenxin-dlut/TransT#2]

**Comment by chenxin-dlut** [comment]
In default settings, it takes about 9 days on two Titan RTX GPUs.
In order to save time, you can reduce the number of epochs and decay the learning rate in advance. 
For example, train 400 epoch and decay the learning rate at 250th epoch. In this way, the training time will be reduced to about 4 days. According to our experience, it will not have much impact on the performance
